plugin/trino-hive/pom.xml

```diff
@@ -463,6 +464,7 @@
                         <configuration>
                             <includes>
                                 <include>**/TestHiveFailureRecoveryFile.java</include>
+                                <include>**/TestHiveFailureRecoveryS3.java</include>
```

Comment:
Lot's 30+mins. Actually we need to improve that.
@arhimondr any idea how to make timeout tests faster. They are needed. As only those really are triggering extra attempt files actually written down. Still it would be great to make them faster.